docs/reference/overview.md

```diff
@@ -29,5 +29,5 @@ Learn about the [Camunda release policy](release-policy.md) with some specific c
 ## Additional resources
 
 - [Regions](regions.md) - When you create a cluster in Camunda 8 SaaS, specify a region for that cluster.
-- [Usage metrics](usage-metrics.md) - Understand the three main usage metrics that have an impact on Camunda 8 pricing.
+- [Usage metrics](/reference/data-collection/usage-metrics.md) - Understand the three main usage metrics that have an impact on Camunda 8 pricing.
```

Comment:
I've updated this link without changing it, but highlighting it here in case you'd like to update this to point to the new data collection page instead, and update the summary text to match.

Reply:
Since this is focused on usage metrics and the commercial side, the usage-metrics page is better